source/content/guides/drupal/drupal-unhosted-composer/15-troubleshooting.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Migrate a Composer Managed Drupal 9 Site from Another Platform
+title: Migrate a Composer Managed Drupal Site from Another Platform
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-unhosted-composer/troubleshooting